pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -254,6 +258,24 @@ func NewKubeGenericRuntimeManager(
 		return nil, ErrVersionNotSupported
 	}
 
+	// Check if runtime supports CDI
+	if utilfeature.DefaultFeatureGate.Enabled(features.DevicePluginCDIDevices) || utilfeature.DefaultFeatureGate.Enabled(features.DynamicResourceAllocation) {
+		minVersions := map[string]string{"containerd": minContainerdCDIVersion, "cri-o": minCRIOCDIVersion}
+		if rversion, ok := minVersions[typedVersion.RuntimeName]; ok {
+			runtimeVersion, err := kubeRuntimeManager.Version(ctx)
+			if err != nil {
+				klog.InfoS("WARNING: can't get runtime version", "error", err)
+			} else {
+				cmp, err := runtimeVersion.Compare(rversion)
+				if err != nil {
+					klog.InfoS("WARNING: can't compare versions", "runtime", typedVersion.RuntimeName, "version", runtimeVersion, "minimal version with CDI support", rversion)
+				} else if cmp < 0 {
+					klog.InfoS("WARNING: runtime doesn't support CDI", "runtime", typedVersion.RuntimeName, "version", runtimeVersion, "minimal version with CDI support", rversion)
+				}
```

Comment:
I think it's necessary to use built-in functions that set the logging level. For example

```go
// The code snippet below uses the klog.V function to specify the logging level
klog.V(4).InfoS("Sandbox does not have metadata", "sandbox", s)
```

This approach will help filter logs based on the warn level and ensure that these warnings are not overlooked.

Reply:
PS: Also, the log level placeholder format may differ from the one provided and be set globally. Therefore, it's better to use functions that set the level.

Reply:
Thank you for the review! 

As for the requested change, I tried to follow the existing code in the same funciton: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kuberuntime/kuberuntime_manager.go#L258 

Note, that this is a constructor code for the `KubeGenericRuntimeManager`, it's called only once. I'd guess this is the reason of using `klog.InfoS` instead of `klog.V(N).InfoS`. `klog.V(N)` is used in the functions that are called more frequently, it seems.

Let me know if you still think I need to use klog.V. I'll update the code.

Reply:
Thanks for the explanation. If such a format is already common in a function, then I think there is no need to fix anything.